server/graphql/v2/interface/Account.js

```diff
@@ -269,6 +269,9 @@ const accountFieldsDefinition = () => ({
       return collective;
     },
   },
+  contributionPolicy: {
```

Comment:
Can we rather add that on `AccountWithContributions`? Unless there's a purpose having this field available on Users/Orgs